ivadomed/utils.py

```diff
@@ -211,11 +216,12 @@ def save_nii(data_lst, z_lst, fname_ref, fname_out, slice_axis, debug=False):
     # Return the orientation that transforms from ras to ref_orientation
     trans_orient = nib.orientations.ornt_transform(ras_orientation, ref_orientation)
     # apply transformation
-    arr_pred_ref_space = nib.orientations.apply_orientation(arr_ras, trans_orient)
+    arr_pred_ref_space = threshold_predictions(nib.orientations.apply_orientation(arr_ras, trans_orient))
```

Comment:
WOuld it be possible to add a param which controls the thresholding? Either a Bool either the threshold value --> config files. For some applications (eg uncertainty computations), we may want to keep non-binarised values.

Reply:
Updated :)!